polaris-react/src/components/ResourceItem/ResourceItem.scss

```diff
@@ -58,11 +58,6 @@
   height: 100%;
 }
 
-.CheckboxLabel {
-  // Pass-through the item alignment to the underlaying label
-  align-items: inherit;
```

Comment:
This isn't doing what is described, instead its setting item alignment to the value of the parent container which is the initial value `start`.
This was previously being overridden by higher specificity `#{$se23}` styles, so this bug slipped through unnoticed.